src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,74 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+import { actions as currentActions } from '../../features/currentTodo';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const [users, setUsers] = useState<User | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```